docs/docker.md

```diff
@@ -2,15 +2,33 @@
 
 ## Overview
 
-Docker can automate much of the configuration for you.
+Docker packages software into virtual containers so it can run on any Linux, Windows, or macOS computer. It can automate much of the configuration for you.
 
-
-## Using docker
+## Using Docker
 
 ### Prerequisites
 
-* Recent versions of `docker` and `docker-compose`.
-* A database dump of the full AGAGD schema, stored at `schema.sql`. (in the root directory of the repository)
+- Recent versions of `docker` and `docker-compose`.
+- A database dump of the full AGAGD schema, stored at `schema.sql`. (in the root directory of the repository)
+
+### [Installation](https://docs.docker.com/engine/install)
+
+#### Mac & Windows
+
+Installing the [Docker Desktop](https://www.docker.com/products/docker-desktop) application is highly recommended.
+
```

Comment:
I know this points to the Docker Desktop Application which is used on both Windows and Mac. However, there are some caveats which should be noted especially for Windows. I think some users who are new to Docker may find a block if they run into any WSL2 issues. This may cause a frustration which some of our developers may not be able to help with do to a Linux based background.